src/WorkOS.net/Services/SSO/Entities/Profile.cs

```diff
@@ -56,6 +56,12 @@ public class Profile
         [JsonProperty("last_name")]
         public string LastName { get; set; }
 
+        /// <summary>
+        /// The user's groups.
+        /// </summary>
+        [JsonProperty("groups")]
+        public string[] Groups { get; set; }
```

Comment:
Using a `List` would be a bit more idiomatic here:

```suggestion
        public List<string> Groups { get; set; }
```